src/SDK/Client/StateTransitionBuilder/getTypeOfRecord.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-ignore
+import Document from "@dashevo/dpp/lib/document/Document";
+// @ts-ignore
+import DataContract from "@dashevo/dpp/lib/dataContract/DataContract";
+// @ts-ignore
+import Identity from "@dashevo/dpp/lib/identity/Identity";
+import {Record, StateTransitionBuilderTypes} from "./StateTransitionBuilder";
+
+const getTypeOfRecord = (record: Record) => {
+    switch (typeof record) {
```

Comment:
When I try using `platform.contracts.broadcast(`, this switch always hits the `default` case and throws an error.

`console.dir({record})` returns
```{ record:
   DataContract {
     contractId: 'At44pvrZXLwjbJp415E2kjav49goGosRF3SB1WW1QJoG',
     version: 1,
     schema: 'https://schema.dash.org/dpp-0-4-0/meta/data-contract',
     documents: { note: [Object] },
     definitions: {} } }
```

`console.log(typeof record)` returns:
`object`

Reply:
Actually looks like the issue might be that `DataContract.prototype.name` used in the `case` is `undefined`

Reply:
Looks like I might have messed this one, thanks for the notice. Need to cover tests here btw. I'm on it

Reply:
Covered there : https://github.com/dashevo/DashJS/pull/33/commits/b49f74b4b8e03e9d1020dd789c62f4310a4fc1ad